addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -122,6 +122,8 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         // grid padding option so it is not applied on inner rows.
         const $gridPaddingOptions = $html.find('[data-css-property="--grid-item-padding-y"], [data-css-property="--grid-item-padding-x"]');
         $gridPaddingOptions.attr("data-apply-to", ".row.o_grid_mode");
+        //TODO adapt in 18.0 as well as in master
+        $html.find("a[role='tab'], a[role='button'], a.btn").attr("data-oe-type", "button");
```

Comment:
```suggestion
        //TODO adapt in 18.0 as well as in master
        $html[0].querySelectorAll("a[role='tab'], a[role='button'], a.btn").forEach((el) => el.setAttribute("data-oe-type", "button"));
```

What we need to adapt in 18.0 and master, would you please add note in details, it will not be needed if we are going to handle this fix with ```oe_unbreakable```

Reply:
For master we directly add `oe_unbreakable` class to each snippets where anchor buttons break.
And going for the fix for 18.0 we need to adapt `_patchForComputeSnippetTemplates()` which handles operations which require XML changes that can be changed in master.